features/nanostack/mbed-mesh-api/mbed-mesh-api/WisunBorderRouter.h

```diff
@@ -21,10 +21,10 @@
  * \brief Struct br_information Border router dynamic information.
  */
 typedef struct ws_br_info {
-    /** Address prefix given to devices in network  set to 0 if not available*/
-    uint8_t ipv6_prefix[8];
-    /** IID of Border router */
-    uint8_t ipv6_iid[8];
+    /** Mesh Interface Global Address */
```

Comment:
These are different things and should be kept as separated

Reply:
Kept it separated.